docs/advanced-usage.rst

```diff
@@ -538,6 +538,37 @@ Here's an example using brotli encoding via the ``Accept-Encoding`` header:
         headers={"Accept-Encoding": "br"}
     )
 
+Zstandard Encoding
+------------------
+
+`Zstandard <https://datatracker.ietf.org/doc/html/rfc8878>`_
+is a compression algorithm created by Facebook with better compression
+than brotli, gzip and deflate (see `benchmarks <https://facebook.github.io/zstd/#benchmarks>`_)
+and is supported by urllib3 if the `python-zstandard <https://pypi.org/project/zstandard/>`_ package is installed.
```

Comment:
Realized thanks to the note below that we should call this package `zstandard` after its name on PyPI.

```suggestion
and is supported by urllib3 if the `zstandard <https://pypi.org/project/zstandard/>`_ package is installed.
```

![image](https://user-images.githubusercontent.com/42327/175360230-bc4d713c-7519-4c3f-85e4-6845e4b5c261.png)